tests/test_normalizing.py

```diff
@@ -1,9 +1,41 @@
 """Test normalizing."""
 from reasoner_util import get_preferred_ids
+from reasoner_util import normalize_qcategories
+from reasoner_util import normalize_predicates
 
 
 def test_get_preferred_ids():
     """Test get_preferred_ids"""
     curies = ["CHEBI:15377", "NCIT:C34373"]  # examples of non-preferred ids
     output = get_preferred_ids(curies)
     assert output == ["PUBCHEM.COMPOUND:962", "MONDO:0004976"]
+
+
+def test_normalize_qcategories():
+    """Test normalize_qcatagories to test strip_descendants with
+    the input of a list of catagories"""
+    catagories = [
+        "biolink:Disease",
+        "biolink:DiseaseOrPhenotypicFeature",
+        "biolink:ThingWithTaxon",
+        "biolink:BiologicalEntity",
+        "biolink:NamedThing",
+        "biolink:Entity",
+      ]
+    output = normalize_qcategories(catagories)
+    assert output == ["biolink:ThingWithTaxon", "biolink:Entity"]
```

Comment:
Note that this assumes that `normalize_qcategories()` does not re-order items.

Reply:
I didn't think that normalize_qcatagories could re-order items. Should I put in a comment about this assumption or should I write the assertion as output == (["biolink:ThingWithTaxon", "biolink:Entity"] or ["biolink:Entity", "biolink:ThingWithTaxon"]) ?

Reply:
I don't have a preference.